types/store/plugins.ts

```diff
@@ -148,6 +149,13 @@ export type PostWillRenderEmbedPluginComponent = {
     toggleable: boolean;
 }
 
+export type FileAttachmentPluginComponent = {
+    id: string;
+    pluginId: string;
+    component: typeof FileAttachment;
```

Comment:
this is only correct if you do not import `FileAttachment` as a type, but as a const instead